lib/foreman_maintain/concerns/base_database.rb

```diff
@@ -4,13 +4,27 @@ module BaseDatabase
       def data_dir
         if el7? && package_manager.installed?('rh-postgresql12-postgresql-server-syspaths')
           '/var/opt/rh/rh-postgresql12/lib/pgsql/data/'
+        elsif debian?
+          deb_psql_data_dir
         else
           '/var/lib/pgsql/data/'
         end
       end
 
+      def deb_psql_data_dir
+        if package_manager.installed?('postgresql-11')
+          '/var/lib/postgresql/11/main/'
+        elsif package_manager.installed?('postgresql-12')
+          '/var/lib/postgresql/11/main/'
+        end
+      end
+
       def postgresql_conf
-        "#{data_dir}/postgresql.conf"
+        if el?
+          "#{data_dir}/postgresql.conf"
+        else
+          '/etc/postgresql/11/main/postgresql.conf'
```

Comment:
```suggestion
          "#{deb_psql_data_dir}/postgresql.conf"
```

Reply:
Contrary to (stupid) Red Hat packaging, Debian actually has it's PostgreSQL config not in data but in etc ;)